src/renderer/helpers/playlists.js

```diff
@@ -31,6 +35,10 @@ export function getSortedPlaylistItems(playlistItems, sortOrder, locale, reverse
   })
 }
 
+export function checkDurationFromVideo(a) {
+  return (isNaN(a.lengthSeconds) || a.lengthSeconds === 0 || typeof a.lengthSeconds === 'string') ? 0 : a.lengthSeconds
```

Comment:
I see this 3 times already just make a helper function?
```suggestion
  return (isNaN(a.lengthSeconds) || a.lengthSeconds === 0 || typeof a.lengthSeconds !== 'number') ? 0 : a.lengthSeconds
```

Reply:
done!